application/akka/scala/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mozilla/sbt:latest
+
+# Install SecretHub
+RUN curl -sLJO https://deb.secrethub.io/amd64
+RUN dpkg -i secrethub-cli-amd64.deb
+RUN rm secrethub-cli-amd64.deb
```

Comment:
Could you put these in a single `RUN` statement?

Reply:
Fixed